Source/Utils/Average.H

```diff
@@ -9,24 +9,19 @@ namespace Average{
 
     /**
      * \brief Returns the cell-centered average of the floating point data contained
-     *        in the input
-     *        <a href="https://amrex-codes.github.io/amrex/doxygen/structamrex_1_1Array4.html">Array4</a>
-     *        \c mf_in_arr.
+     *        in the input Array4 \c mf_in_arr.
      *
      * \param[in] mf_in_arr floating point data to be averaged
      * \param[in] stag      staggering (index type) of the data
-     * \param[in] i         index along x to access the
-     *                      <a href="https://amrex-codes.github.io/amrex/doxygen/structamrex_1_1Array4.html">Array4</a>
-     *                      \c mf_in_arr containing the data to be averaged
-     * \param[in] j         index along y to access the
-     *                      <a href="https://amrex-codes.github.io/amrex/doxygen/structamrex_1_1Array4.html">Array4</a>
-     *                      \c mf_in_arr containing the data to be averaged
-     * \param[in] k         index along z to access the
-     *                      <a href="https://amrex-codes.github.io/amrex/doxygen/structamrex_1_1Array4.html">Array4</a>
-     *                      \c mf_in_arr containing the data to be averaged
-     * \param[in] comp      index along the fourth component of the
-     *                      <a href="https://amrex-codes.github.io/amrex/doxygen/structamrex_1_1Array4.html">Array4</a>
-     *                      \c mf_in_arr containing the data to be averaged
+     * \param[in] i         index along x to access the Array4 \c mf_in_arr containing
+     *                      the data to be averaged
+     * \param[in] j         index along y to access the Array4 \c mf_in_arr containing
+     *                      the data to be averaged
+     * \param[in] k         index along z to access the Array4 \c mf_in_arr containing
+     *                      the data to be averaged
+     * \param[in] comp      index along the fourth component of the Array4 \c mf_in_arr
+     *                      containing the data to be averaged
+     *
      * \return averaged field at cell (i,j,k)
      */
```

Comment:
Reason for cleaning up this Doxygen lines: the web links I added in an earlier PR made the comments non-readable in the code itself, hence I removed them. These changes are not strictly related to this PR, but most of this file is being modified anyways and I'm taking the opportunity to make the Doxygen documentation uniform among the functions defined in this whole new namespace.